config/eslint-config-carbon/plugins/jest.js

```diff
@@ -8,7 +8,8 @@
 'use strict';
 
 module.exports = {
-  plugins: ['eslint-plugin-jest'],
+  plugins: ['eslint-plugin-jest', 'jest-dom'],
+  extends: ['plugin:jest-dom/recommended'],
   overrides: [
     {
```

Comment:
Maybe move extends to overrides scope...

```
  overrides: [
    {
      files: ['*.stories.js', '*-story.js'],
      extends: ['plugin:jest-dom/recommended']

```

Reply:
Yeah I think moving the `extends` to the `overrides` scope is a good idea so this recommended rule set is scoped to files only with those prefixes.